cmd/devicesimulator/main.go

```diff
@@ -47,6 +47,7 @@ func main() {
 	initialDeviceIndex := pflag.Int("initial-device-index", 0, "starting index for device name suffix, (e.g., device-0000 for 0, device-0200 for 200))")
 	metricsAddr := pflag.String("metrics", "localhost:9093", "address for the metrics endpoint")
 	stopAfter := pflag.Duration("stop-after", 0, "stop the simulator after the specified duration")
+	logLevel := pflag.StringP("v", "v", "debug", "logger verbosity level (one of \"fatal\", \"error\", \"warn\", \"warning\", \"info\", \"debug\")")
```

Comment:
If you want to use StringP which gives you a long a and a short flag should we make those different?
```suggestion
	logLevel := pflag.StringP("log-level", "v", "debug", "logger verbosity level (one of \"fatal\", \"error\", \"warn\", \"warning\", \"info\", \"debug\")")
```

Reply:
just wanted to give an option to supply `--v` and `-v`... but your version is also quite good. changing